apps/backoffice-v2/src/pages/TransactionMonitoringAlerts/components/AlertsTable/hooks/useAlertsTableLogic/useAlertsTableLogic.tsx

```diff
@@ -91,7 +91,7 @@ export const useAlertsTableLogic = ({ data }: { data: TAlertsList }) => {
   const { pathname, search } = useLocation();
   const url = `${pathname}${search}`;
   const onRowClick = useCallback(() => {
-    sessionStorage.setItem('transaction-monitoring:transactions-drawer:previous-path', url);
+    // sessionStorage.setItem('transaction-monitoring:transactions-drawer:previous-path', url);
```

Comment:
This is a breaking change. You can introduce a different session storage key if needed.